### PR TITLE
Cleanup 2DC code

### PR DESCRIPTION
**Pull request by yux0**
**What changed?**
Cleanup 2DC code

**Why?**
The code is deprecated

**How did you test it?**
rely on existing tests

**Potential risks**
The conflict resolution breaks

### CODE REVIEW

File: service/history/execution/mutable_state_util.go [hunk 1]
```diff
@@ -338,7 +338,7 @@ func CreatePersistenceMutableState(ms MutableState) *persistence.WorkflowMutable
 		SignalInfos:         signalInfos,
 		RequestCancelInfos:  cancellationInfos,
 		ChildExecutionInfos: childInfos,
-		ReplicationState:    replicationState,
+		VersionHistories:    versionHistories,
```
Comment (yycptt) [inline service/history/execution/mutable_state_util.go:341]: Just curious. How did the code work previously without persisting this field?

File: common/persistence/cassandra/cassandraPersistenceUtil.go [hunk 2]
```diff
@@ -670,8 +578,8 @@ func updateExecution(
 	executionInfo.LastUpdatedTimestamp = time.Unix(0, p.DBTimestampToUnixNano(cqlNowTimestampMillis))
 
 	completionData, completionEncoding := p.FromDataBlob(executionInfo.CompletionEvent)
-	if replicationState == nil && versionHistories == nil {
-		// Updates will be called with null ReplicationState while the feature is disabled
+	if versionHistories == nil {
+		// Updates will be called with null version historyies while the feature is disabled
```
Comment (yycptt) [inline common/persistence/cassandra/cassandraPersistenceUtil.go:582]: nit: `histories`

**Review by yycptt — APPROVED** [review]
LGTM

File: service/history/execution/mutable_state_util.go (hunk 1 above)
Comment (yux0) [reply service/history/execution/mutable_state_util.go:341]: I think this method only used in test and test are mocking the replication state

File: common/persistence/cassandra/cassandraPersistenceUtil.go (hunk 2 above)
Comment (yux0) [reply common/persistence/cassandra/cassandraPersistenceUtil.go]: fixed the typo

**Pull request merged**